orm/model_bucket_test.go

```diff
@@ -0,0 +1,174 @@
+package orm
+
+import (
```

Comment:
In general tests give good basic coverage.
I would say more in-depth would be better, but since 90% of the issues are covered in orm, it is not as important.

However, it would be good to do some error checking with the wrong model class being used.

Reply:
Done in https://github.com/iov-one/weave/pull/615/commits/3d3a25c4ae98f039a95c888c596c0e6cfe483ab0

I believe this validation should be done in the original bucket implementation. It is not and there is clean access to the model passed to the bucket constructor. That is why I had to use [the power of guessing™](https://github.com/iov-one/weave/pull/615/commits/3d3a25c4ae98f039a95c888c596c0e6cfe483ab0#diff-2aaeb20d3a67378341e1ea83d6efba11R198) to be able to check types. 

I have decided to write it like this only because we want to update model bucket implementation to rely on the original bucket and then getting model type will be clean and easy operation :crossed_fingers: